tools/openocd/modm/stm32h7xx_devebox.cfg

```diff
@@ -0,0 +1,10 @@
+# Cheap STM32F4x1 "Mini-F4x1" Minimum System Development Board
+
+# Must be specified by the user via `modm:build:openocd.cfg` option
+#source [find interface/stlink-v2.cfg]
```

Comment:
Please copy the programming description and warning like this, otherwise people might get frustrated:
https://github.com/modm-io/modm/blob/develop/src/modm/board/blue_pill_f103/module.lb#L82-L84

Reply:
fixed.